arrows/core/associate_detections_to_tracks_threshold.cxx

```diff
@@ -102,7 +100,8 @@ ::check_configuration( VITAL_UNUSED vital::config_block_sptr config ) const
 bool
 associate_detections_to_tracks_threshold
 ::associate( kwiver::vital::timestamp ts,
-             kwiver::vital::image_container_sptr /*image*/,
+             kwiver::vital::image_container_sptr //image
+,
```

Comment:
Should the comma be on a separate line?

Reply:
Good catch. Thanks. I've fixed this and will commit it shortly.